Framework/DataObjects/test/MDFramesToSpecialCoordinateSystemTest.h

```diff
@@ -27,7 +27,7 @@ class MDFramesToSpecialCoordinateSystemTest : public CxxTest::TestSuite {
 public:
   void test_that_throws_for_non_md_workspace() {
     // Arrange
-    const std::shared_ptr<MatrixWorkspace> ws =
+    const std::shared_ptr<MatrixWorkspace> &ws =
```

Comment:
Did `cppcheck` warning on this as I don't think it makes sense. `std::make_shared` creates a brand new one so there is no other object to reference here I don't think?

Reply:
Thanks, yes the regex picked up some cases like this which I have been working to remove so will continue 👍